src/SavingsDai.sol

```diff
@@ -76,6 +76,7 @@ contract SavingsDai {
     event Transfer(address indexed from, address indexed to, uint256 value);
     event Deposit(address indexed sender, address indexed owner, uint256 assets, uint256 shares);
     event Withdraw(address indexed sender, address indexed receiver, address indexed owner, uint256 assets, uint256 shares);
+    event Referral(uint16 indexed referral, address indexed owner, uint256 assets, uint256 shares);
```

Comment:
I'm using the same variable name as in `Deposit` where the recipient of the shares is called `owner`. I know the function variable representing the owner is called `receiver` but that follows the same naming convention as in https://ethereum.org/en/developers/docs/standards/tokens/erc-4626/

Reply:
Yeah, was about to say that erc-4626 has difference in the names for that between the function parameter and the event parameter. Since this is an event I'd probably sticj with `owner`.

Reply:
Ok yeah if the standard is all fucked up let's live with it.

Reply:
You can resolve it!